terraform/s3.tf

```diff
@@ -1,10 +1,21 @@
+locals {
+  pcc-reporting-bucket-name = "pcc-recon-reports-${var.target_env}"
```

Comment:
we also have the "pcc-integration-data-files-${var.target_env}" bucket (we drop the files from our sftp server via aws-xfer family here). I am not super clear on how locals work in terraform but I just wanted to make sure that we didn't need to include this bucket as well?

Reply:
@chelsea-EYDS  I set up the logging of that bucket separately (in sftp.tf) so, should not be needed here